Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,45 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
```

Comment:
This is what a sample comment would look like in a PR.

Reply:
moment, Do you mean that the comment from me was good or do I have to change something next time to my PR

Reply:
Everything looks good to me.